clang/test/CIR/IR/invalid.cir

```diff
@@ -792,6 +792,76 @@ cir.func @const_type_mismatch() -> () {
 
 // -----
 
+!u16i = !cir.int<u, 16>
```

Comment:
This is awesome, thanks for adding lots of invalid checks. Can you please add a new `.cir` file with a simple use case of all introduced op/type/attr that tests the parser functionality?

Reply:
Updated.